addOns/pscanrulesBeta/src/test/java/org/zaproxy/zap/extension/pscanrulesBeta/InformationDisclosureSuspiciousCommentsUnitTest.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesBeta;
+
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import java.net.URL;
+
+import org.apache.commons.httpclient.URI;
+import org.apache.commons.httpclient.URIException;
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMalformedHeaderException;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+
+public class InformationDisclosureSuspiciousCommentsUnitTest
+        extends PassiveScannerTest<InformationDisclosureSuspiciousComments> {
+	
+    private HttpMessage msg;
+    
+    @Override
+    protected InformationDisclosureSuspiciousComments createScanner() {
+    	return new InformationDisclosureSuspiciousComments();
+    }
+    
+	    protected HttpMessage createHttpMessageWithRespBody(String responseBody,
```

Comment:
This method seems to be indented an extra level.

The file also seems to use tabs not spaces. Running the grade `spotlessApply` task should correctly format things.

Reply:
It'd fix the wonky indent on the `body` declarations in the test methods below as well.